### PR TITLE
clean up PMD configuration

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Change Log
```
Comment (malaffoon) [inline CHANGELOG.md]: We should discuss this. In pretty much every project i've ever worked on, a CHANGELOG file included in the source was used to capture changes: new features, fixed bugs, etc. However, i notice that, for SmarterBalanced, we've been using Confluence combined with JIRA queries to produce the equivalent information. We shouldn't do both. I added this just to trigger the discussion. @rmitchell3 @millerts

**Comment by malaffoon** [comment]
Pushing onwards, will address any concerns.

File: CHANGELOG.md (hunk 1 above)
Comment (millerts) [reply CHANGELOG.md]: I don't think we'll be able to get around the "How should features work" unless we want to duplicate (less in depth) here.

Comment (malaffoon) [reply CHANGELOG.md]: Oh, this wouldn't replace the documentation associated with the feature work itself. I agree that that is separate and necessary thing. But the list of all fixed bugs, a brief note for each new feature, etc. would be nice to put in here. Or not. We can decide as we go along.

**Pull request merged**